openmetadata-ui/src/main/resources/ui/src/components/CreateUser/CreateUser.component.tsx

```diff
@@ -515,79 +521,79 @@ const CreateUser = ({
         return (
           <>
             <Form.Item
-              label="PrivateKey"
+              label={t('label.privateKey')}
               name="privateKey"
               rules={[
                 {
                   required: true,
-                  message: 'PrivateKey is required',
+                  message: t('message.privateKey-required'),
                 },
               ]}>
               <Input.Password
                 data-testid="privateKey"
                 name="privateKey"
-                placeholder="privateKey"
+                placeholder={t('label.privateKey')}
                 value={ssoClientConfig?.privateKey}
                 onChange={handleOnChange}
               />
             </Form.Item>
             <Form.Item
-              label="ClientId"
+              label={t('label.client-id')}
               name="clientId"
               rules={[
                 {
                   required: true,
-                  message: 'ClientId is required',
+                  message: t('message.client-id-required'),
                 },
               ]}>
               <Input
                 data-testid="clientId"
                 name="clientId"
-                placeholder="clientId"
+                placeholder={t('label.client-id')}
                 value={ssoClientConfig?.clientId}
                 onChange={handleOnChange}
               />
             </Form.Item>
             <Form.Item
-              label="OrgURL"
+              label={t('label.org-url')}
               name="orgURL"
               rules={[
                 {
                   required: true,
-                  message: 'OrgURL is required',
+                  message: t('message.org-url-required'),
                 },
               ]}>
               <Input
                 data-testid="orgURL"
                 name="orgURL"
-                placeholder="orgURL"
+                placeholder={t('label.org-url')}
                 value={ssoClientConfig?.orgURL}
                 onChange={handleOnChange}
               />
             </Form.Item>
             <Form.Item
-              label="Email"
+              label={t('label.email')}
               name="oktaEmail"
               rules={[
                 {
                   required: true,
                   type: 'email',
-                  message: 'Service account Email is required',
+                  message: t('message.service-email-required'),
```

Comment:
Let's replace it with the below localisation keys
Below are already present in JSON file
```
"field-required": "{{field}} is required",
"field-required-plural": "{{field}} are required",

```